rules/linux/lnx_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,31 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+    - https://attack.mitre.org/techniques/T1016
+logsource:
+    product: unix
+detection:
+    keywords:
+        # Linux Only
+        - 'arp -a'
+        - 'ip'
+        - 'ss'
+        # macOS and Linux
+        - 'netstat'
+        - 'ifconfig'
+        # macOS only
+        - 'defaults read /Library/Preferences/com.apple.alf'
+        - 'socketfilterfw'
+    condition: keywords
```

Comment:
I would recommend you use "commandline" and "ProcessName" here, just like we do it for windows events.

Reply:
thanks for the guidance, I have incorporated your recommendations into the latest commit.

Reply:
Thank you!